base/show.jl

```diff
@@ -1699,10 +1699,32 @@ function show_unquoted(io::IO, val::SSAValue, ::Int, ::Int)
     if get(io, :maxssaid, typemax(Int))::Int < val.id
         # invalid SSAValue, print this in red for better recognition
         printstyled(io, "%", val.id; color=:red)
+    elseif haskey(io, :compacted) && !io[:compacted]
+        # in the uncompacted part of IncrementalCompact, SSAValues refer to old values
+        printstyled(io, "%", val.id; color=:yellow)
     else
         print(io, "%", val.id)
     end
 end
+
+function show_unquoted(io::IO, val::Core.Compiler.OldSSAValue, ::Int, ::Int)
+    if haskey(io, :compacted) && io[:compacted]
```

Comment:
```suggestion
    if get(io, :compacted, false)::Bool
```

Reply:
This was intentional, to keep the old rendering when not working with IncrementalCompact.